Makefile

```diff
@@ -1,5 +1,8 @@
 BUILD_TAG ?= bbb-release
-OUTPUT_DIR ?= build/$(BUILD_TAG)
+include $(BUILD_DIR)/makefile.d/base.mk
+
+# Fall back to OUTPUT_DIR. Keeps this build working if there is a mismatch with the xl repo.
+#LIBRARY_OUTPUT_DIR ?= $(OUTPUT_DIR)
```

Comment:
This line is meant to be commented out?

Reply:
Bah...

Reply:
See https://github.com/Vorne/pru-driver/pull/8/commits/bfe5d853ee873fe2f6323b2d2aa9d38d6b344448